builtin/logical/aws/backend_test.go

```diff
@@ -195,6 +196,10 @@ func TestBackend_throttled(t *testing.T) {
 }
 
 func testAccPreCheck(t *testing.T) {
+	if !hasAWSCredentials() {
+		t.Skip("The default config for AWS could not be resolved; see https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials for instructions.")
```

Comment:
I think we can probably make this a bit more clear. Default config for AWS seems a bit vague. Perhaps something along the lines of `Skipping because we could to resolve AWS credentials from the environment. See <href> for information on how to set up AWS credentials.`